lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -693,15 +702,20 @@ def make_axes_locatable(axes):
     return divider
 
 
-def make_axes_area_auto_adjustable(ax,
-                                   use_axes=None, pad=0.1,
-                                   adjust_dirs=None):
+def make_axes_area_auto_adjustable(
+        ax, use_axes=None, pad=0.1, adjust_dirs=None):
+    """
+    Add auto-adjustable padding around *ax* to take its decorations (title,
+    labels, ticks, ticklabels) into account during layout, using
+    `Divider.add_auto_adjustable_area`.
+
+    By default, the decorations of *ax* are taken into account, but the
+    decorations of other artists can be used instead using *use_axes*.
```

Comment:
```suggestion
    By default, the padding is determined from the decorations of *ax*.
    Pass *use_axes* to consider the decorations of other Axes instead.
```
Mainly to get rid of the double use of "use" :stuck_out_tongue_closed_eyes: .